novice-goals.Rmd

```diff
@@ -102,6 +102,46 @@ but nothing more.
 
 -   How can I represent and manage missing values?
     -   NA
+    Python Learning Objectives
+    -   Identify missing values (e.g., NaN) in your datasets
+    -   Examine the impact of missing valus on your datasets
+    -   Demonstrate and use commands that process missing values
+    -   Demonstrate how to replace missing values
+
+    Python Lecture Notes (general info) 
+    -   Missing Completely at Random (MCAR)
+    -   Missing at Random (MAR)
+    -   Missing Not at Random (MNAR)
+    -   Missing data at scale (what's a little and what's alot?)
+    -   Imputation and Multiple Imputation 
+    -   When summing the data, missing values will be treated as zero
+    -   If all values are missing, the sum will be equal to NaN
+    -   cumsum() and cumprod() methods ignore missing values but preserve them in the resulting arrays
+    -   Missing values in GroupBy method are excluded (just like in R)
```

Comment:
Potentially worth mentioning that using `.groupby()` with [`observed=True` includes NaNs for categoricals](https://pandas.pydata.org/pandas-docs/stable/whatsnew/v0.23.0.html#new-observed-keyword-for-excluding-unobserved-categories-in-groupby).

Reply:
Yeah, I contemplated adding .groupby() too. If it's not included elsewhere, it'll be added here.